boat_tutorials/conceptual/target_strength.md

```diff
@@ -1,3 +1,62 @@
-# Target Strength
+# Target strength
 
-Coming soon!
+
+## Introduction
+- intuitively, what is target strength (TS)? 
+    - how much sound energy bounces off (scatters) from a target? 
+absorption
+    - more generally: scattering strength?
+- when/how is it used?
+    - characterize scatterers or scattering sources independent of the complications of the transmit signals ([connec to SL]()) and the propagation environment ([connect to TL]())
+
+
+## Setup and terminology
+- monostatic vs bistatic ([deeper topic]())
+- differential scattering cross section --> TS ([connect to decibel page]())
+- total scattering cross section
+- extinction cross section (total power removed from the incident sound):
+    - scattering cross section + absorption cross section
+
+## Discrete scatterers
+- what can we infer using these models?
+    - fish/zooplankton abundance
+    - density of suspended sediment ([deeper topic]())
+- point scatterer
+- spherical cows
+    - rigid sphere
+    - bubble
+    - elastic sphere ([deeper topic]())
+- non-spherical cows
+    - bring in orientation ([connect to SL/beampattern]()) and frequency dependency ([connect to spectrum]())
+    - types:
+        - air inclusion (fish swimbladder)
+        - soft body (squid, krill)
+        - elastic structure (pteropod)
+- practical example:
+    - fish school: assume no bio attenuation, no multiple scattering, no shadowing
+    - echogram with fish vs zooplankton
+
+
+## Surface scattering
+- what can we infer using these models?
+    - seabed composition
+    - surface roughness
+- rigid surface vs rough interface
+    - angular and frequency dependency ([connect to spectrum/fourier analysis]())
```

Comment:
- plane wave assumption
- definition: scattering per unit area
- connect to real world examples
- perfect place to talk about sidescan: find object from noisy seafloor
- concept applies to sea surface scattering